### PR TITLE
Add IntesisHome climate platform documentation

### PR DESCRIPTION
**Pull request by jnimmo**
**Description:**
Add documentation for IntesisHome climate platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#25364

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/climate.intesishome.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  description: Your password for IntesisHome
+  required: true
+  type: string
+{% endconfiguration %}
+
+This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
+By default the component will be named using the friendly device name from the IntesisHome website or application.
+If internet connectivity is lost the device will be marked as unavailable after 5 minutes.
+
+### {% linkable_title Supported services %}
```
Comment (frenck) [inline source/_components/climate.intesishome.markdown]:
```suggestion
### Supported services
```

File: source/_components/climate.intesishome.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+logo: intesishome.png
+ha_category: Climate
+ha_release: 0.96
+ha_iot_class: Cloud Push
+---
+
+The `IntesisHome` climate platform lets you control [IntesisHome](https://www.intesishome.com) devices via the IntesisHome cloud. IntesisHome provides integrations with air conditioners including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
+
+Note: IntesisHome products are a separate product line to IntesisBox. This platform does not support IntesisBox devices (which can be controlled locally using the WMP protocol).
+
```
Comment (frenck) [inline source/_components/climate.intesishome.markdown]:
```suggestion

## Configuration

```

File: source/_components/climate.intesishome.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  type: string
+password:
+  description: Your password for IntesisHome
+  required: true
+  type: string
+{% endconfiguration %}
+
+This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
+By default the component will be named using the friendly device name from the IntesisHome website or application.
+If internet connectivity is lost the device will be marked as unavailable after 5 minutes.
```
Comment (frenck) [inline source/_components/climate.intesishome.markdown]:
```suggestion
If internet connectivity is lost, the device will be marked as unavailable after 5 minutes.
```

File: source/_components/climate.intesishome.markdown [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  required: true
+  type: string
+password:
+  description: Your password for IntesisHome
+  required: true
+  type: string
+{% endconfiguration %}
+
+This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
+By default the component will be named using the friendly device name from the IntesisHome website or application.
```
Comment (frenck) [inline source/_components/climate.intesishome.markdown]:
```suggestion
By default, the component will be named using the friendly device name from the IntesisHome website or application.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @jnimmo, a couple of small changes and we are good to go! 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — CHANGES_REQUESTED** [review]
Sorry, @klaasnicolaas Just pointed me to another issue.
The filename is:
`source/_components/climate.intesishome.markdown`

This should be:
`source/_components/intesishome.markdown`

Could you please adjust that?

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/intesishome.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  type: string
+{% endconfiguration %}
+
+This component opens a TCP connection with the IntesisHome API to receive temperature and status updates, and to issue commands.
+By default, the component will be named using the friendly device name from the IntesisHome website or application.
+If internet connectivity is lost, the device will be marked as unavailable after 5 minutes.
+
+### Supported services
+
+Available services:
```
Comment (klaasnicolaas) [inline source/_components/intesishome.markdown]: What you could do is a small description of what each service does, but that is extra and not something we demand 😄

**Comment by frenck** [comment]
Rebased PR onto the latest `next` branch and moved the page to the new location in our documentation repository (`_integrations` folder).

**Review by frenck — APPROVED** [review]

**Pull request merged**